apps/spruce/src/pages/adminSettings/tabs/GeneralTab/FeatureFlagsTab/formSchema.ts

```diff
@@ -0,0 +1,149 @@
+import { GetFormSchema } from "components/SpruceForm";
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import widgets from "components/SpruceForm/Widgets";
+
+const serviceItems: Record<string, string> = {
+  taskDispatchDisabled: "Dispatch tasks",
+  largeParserProjectsDisabled: "Large parser projects",
+  hostInitDisabled: "Create and provision hosts",
+  podInitDisabled: "Create and provision pods",
+  monitorDisabled: "Monitor hosts and tasks",
+  agentStartDisabled: "Start agents on hosts",
+  schedulerDisabled: "Schedule tasks",
+  hostAllocatorDisabled: "Host Allocator",
+  systemFailedTaskRestartDisabled: "Auto-restart system failures",
+  podAllocatorDisabled: "Allocate pods for container tasks",
+  unrecognizedPodCleanupDisabled: "Clean up unrecognized pods",
+  cloudCleanupDisabled: "Cloud Provider Cleanup",
+  taskReliabilityDisabled: "Task Reliability",
```

Comment:
I don't see this one on the current admin settings. Maybe we can delete it

Reply:
sgtm